Dockerfile.guardrails

```diff
@@ -0,0 +1,34 @@
+ARG REGISTRY
+FROM ${REGISTRY}/ubi8/go-toolset:1.18.10-1 AS builder
+ARG GATEKEEPER_VERSION
+ENV DOWNLOAD_URL=https://github.com/open-policy-agent/gatekeeper/archive/v${GATEKEEPER_VERSION}.tar.gz
+
+ARG BUILDKIT_SBOM_SCAN_STAGE=true
+
+ENV GO111MODULE=on \
+    GOOS=linux \
+    GOPATH=/go/ \
+    GOARCH=amd64 \
+    CGO_ENABLED=0
+
+WORKDIR ${GOPATH}/src/github.com/open-policy-agent/gatekeeper
+USER root
+RUN yum update -yq
```

Comment:
We don't actually want to do this, since it updates the go-toolset to 1.19 -- we should do it in the runtime container still, tho.

Reply:
Done, made changes.